jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/definition/details/multi/advance/AdvancedProcessDefDetailsMultiPresenter.java

```diff
@@ -40,6 +41,9 @@
     @Inject
     private AdvancedViewProcessDefDetailsPresenter detailPresenter;
 
+    @Inject
+    private ProcessDiagramPresenter processDiagramPresenter;
```

Comment:
If we removed _Process_ part from _Process Diagram_ in the tab, I'm wondering does it make sense to do the same in the code itself, so that we keep consistency between UI and the client code.

Reply:
@cristianonicolai Yeah, I get your point. Just figured throwing it out there motivated by our conversation during today's meeting with discrepancy between dashboards and reporting, but that is far more drastic example than this here, of course :)